src/everysinglestreet.py

```diff
@@ -0,0 +1,39 @@
+import gpxpy
+
+class GPSPoint:
+    def __init__(self, lat:float, lon:float, elevation, timestamp):
+        self.lat = lat
+        self.lon = lon
+        self.elevation = elevation
+        self.timestamp = timestamp
+
+    def print_point(self):
+       print('{} {} at time {} at elevation {}'.format(self.lat, self.lon, self.timestamp, self.elevation))
+
+
+class Ride:
+    def __init__(self, filename):
+        self.filename = filename
+        self.GPSPoints, self.rideDate = self.parseGPXFile()
+
+    def parseGPXFile(self):
```

Comment:
make this function private

Reply:
Yeah good call need to lead this with an underscore